buildSrc/src/main/java/Config.kt

```diff
@@ -115,6 +115,7 @@ object Config {
 
         val graphQlJava = "com.graphql-java:graphql-java:17.3"
         val graphQlJava22 = "com.graphql-java:graphql-java:22.1"
+        val graphQlJavaNew = "com.graphql-java:graphql-java:24.0"
```

Comment:
Should we go with major versions for the name like before? I.e. `graphQlJava24`?

Reply:
I thought about it but think it's better to skip the version in the name since we probably want to keep this aligned with Spring Boots required version in the future.

Or it would be even better to have Spring Boot take care of handling the version via its gradle plugin.